crates/interface-types/tests/custom_signal_handler.rs

```diff
@@ -0,0 +1,273 @@
+use core::cell::Ref;
```

Comment:
This is a good test, but it's not really specific to interface types. Would it makes sense to move it to wasmtime's top-level tests directory?